wsclient/src/main/java/org/hpccsystems/ws/client/platform/WorkunitInfo.java

```diff
@@ -426,7 +591,7 @@ public WorkunitInfo setOriginalEclWatchUrl(String serverhost) {
         return raw;
     }
 	
-	public WorkunitInfo setNamedValues(NamedValue[] raw) 
+	public WorkunitInfo setNamedValues(org.hpccsystems.ws.client.gen.wsworkunits.v1_73.NamedValue[] raw) 
```

Comment:
if we have multiple versions of namedvalue, does it warrant creating a wrapper for it?